src/ssh.act

```diff
@@ -37,26 +40,36 @@ actor Client(cap: net.TCPConnectCap,
 #
 #    def _connect(c):
 #        NotImplemented
-#
+
 
 # TODO: implement support for channels
 # AFAIK, all things over ssh are done via channels, so need some channel
 # primitive, maybe an actor per channel that then multiplexes into the Client
 # session? Prolly need some higher level wrappers for common things like
 # starting a shell or running a single command. SFTP / SCP would be nice too,
 # but for sometime in the future. Custom subsystems need to be supported too.
+actor Channel(cap: Client):
```

Comment:
```suggestion
actor Channel(client: Client):
```
we take the client, not a capability reference, so using a better name for the argument

Reply:
done